snipsTools.py

```diff
@@ -1,45 +1,21 @@
-import io
-import configparser
-
-CONFIGURATION_ENCODING_FORMAT = "utf-8"
-
-
-class SnipsConfigParser(configparser.SafeConfigParser):
-    def to_dict(self):
-        return {
-            section: {
-                optname: option for optname, option in self.items(section)
-            } for section in self.sections()
-        }
+import toml
 
+class SnipsConfigParser:
     @staticmethod
     def read_configuration_file(configuration_file):
         try:
-            with io.open(
-                    configuration_file,
-                    encoding=CONFIGURATION_ENCODING_FORMAT) as f:
+            return toml.load(configuration_file)
 
-                conf_parser = SnipsConfigParser()
-                conf_parser.readfp(f)
-                return conf_parser.to_dict()
-
-        except (IOError, configparser.Error) as e:
+        except (OSError, ValueError) as e:
```

Comment:
the exception thrown when the file does not exist is a FileNotFoundError and not an IOError in python3 (both IOError and FileNotFoundError are a OSError -> catch OSError or valueError when the format is wrong)

toml can read ini files aswell, since they have the same format with a couple less functionalities